### ISSUE TITLE
FS type bcachefs is Unknown

### ISSUE THREAD

**Issue by kezyr** [archlinux/archinstall#2474]
Hi, when I select Pre-mounted configuration for Disk configuration where rootfs is bcachefs then it shows FS type as Unknown. 
![image](https://github.com/archlinux/archinstall/assets/62434319/b15852b8-3180-42ac-afd7-ed2a00dc87cc)

I use Hyper-V with 1 disk, latest version of archinstall.
I formated sda2 as bcachefs and mounted to /mnt, and sda1 as vfat and mounted to /mnt/efi

Then the installer craches because of fstype not set

**Comment by svartkanin** [comment]
Bcachefs isn't something parted supports which is being used for partitioning under the hood.

**Comment by codefiles** [comment]
> Bcachefs isn't something parted supports which is being used for partitioning under the hood.

That is not accurate for two reasons:

1. The user is attempting an install with a pre-mounted configuration and therefore has already partitioned and formatted.
2. [parted(8)](https://man.archlinux.org/man/extra/parted/parted.8.en) states that providing a file system when creating a partition is optional. This is true of pyparted as well, give it a try.

I will submit a pull request to fix this since it should work.

**Comment by svartkanin** [comment]
Apologies I misread the issue, I thought it was for manual partitioning. In that case yes, pre-mounted should work and it's a bug.
However, the pre-mounted feature is in general somewhat limited still as it doesn't support many detections, btrfs and lvm being some of them.

**Comment by codefiles** [comment]
> However, the pre-mounted feature is in general somewhat limited still as it doesn't support many detections, btrfs and lvm being some of them.

Btrfs works with the pre-mounted feature in the latest version.

**Comment by dathide** [comment]
This issue is still occurring for me

**Comment by Torxed** [comment]
Even after the release yesterday?

**Comment by kezyr** [comment]
yes the issue is still here (checked using archlinux-2024.09.01-x86_64.iso)
During install there is an error message "File system type not set".

**Comment by avnyu** [comment]
https://github.com/archlinux/archinstall/blob/957fa06ade3e6ba144b96a1ff2796039e6821396/archinstall/lib/installer.py#L1042
removing this line fix the issue for me, and without root fs type my desktop still boot normally
